test/EndToEnd/GivenPortableApps.cs

```diff
@@ -0,0 +1,87 @@
+﻿using System;
+using System.Collections.Generic;
+using System.IO;
+using System.Linq;
+using System.Xml.Linq;
+using FluentAssertions;
+using Microsoft.DotNet.TestFramework;
+using Microsoft.DotNet.Tools.Test.Utilities;
+using NuGet.ProjectModel;
+using NuGet.Versioning;
+using Xunit;
+
+namespace EndToEnd
+{
+    public class GivenPortableApps : TestBase
+    {
+        [Theory]
+        [ClassData(typeof(SupportedNetCoreAppVersions))]
+        public void ItDoesNotRollsForwardToTheLatestVersion(string minorVersion)
```

Comment:
@dsplaisted does this test looks good? And I need to keep work on Generate the flag `UseBundledVersionAsDefaultVersion` next

Reply:
Grammar nit: rename the method to `ItDoesNotRollForwardToTheLatestVersion` ("roll" instead of "rolls").